prism/src/main/java/org/apache/falcon/resource/proxy/SchedulableEntityManagerProxy.java

```diff
@@ -310,6 +319,12 @@ public APIResult update(
             @Dimension("colo") @QueryParam("colo") String ignore,
             @QueryParam("skipDryRun") final Boolean skipDryRun) {
 
+        try {
+            isEntityPartOfAnExtension(EntityUtil.getEntity(type, entityName));
```

Comment:
Shouldn't you use the supplied entity definition here, rather than fetching it from from Config Store?